### PR TITLE
Increase max_length values for Notification model fields

### PR DESCRIPTION
**Pull request by jordanmkoncz**
This fixes https://github.com/worthwhile/django-herald/issues/46.

Note that I also increased the `max_length` for the `Notification.verbose_name` field, as I assume there wasn't a reason to have a tight restriction on that field either.

### CODE REVIEW

File: CHANGELOG.txt [hunk 1]
```diff
@@ -1,3 +1,6 @@
+=== 0.2.1 (unreleased) ===
+- Changed Notification.notification_class and Notification.verbose_name fields to have less restrictive `max_length`
```
Comment (jproffitt) [inline CHANGELOG.txt]: @jordanmkoncz we actually haven't released 0.2 yet, so would you mind just moving this down into the 0.2 notes?

File: herald/__init__.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 """
 Notification classes. Used for sending texts and emails
 """
-__version__ = '0.2'
+__version__ = '0.2.1'
```
Comment (jproffitt) [inline herald/__init__.py]: Since we haven't release 0.2 yet, we can just add this to the 0.2 release. Thanks!

**Comment by jordanmkoncz** [comment]
@jproffitt no worries, that makes sense. I've amended the commit with your requested changes.

**Comment by jproffitt** [comment]
Thanks @jordanmkoncz !

**Pull request merged**